buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -916,6 +1010,18 @@
     "DockerCommonRunArgs_Debian8": {
       "value": "--name $(PB_DockerContainerName)$(DockerTag_Debian8) -v \"$(PB_SourcesDirectory):$(PB_GitDirectory)\" -v $(Build.StagingDirectory)/sharedFrameworkPublish/:/root/sharedFrameworkPublish/ -w=\"$(PB_GitDirectory)\" $(DockerImageName_Debian8)"
     },
+    "DockerTag_Debian9": {
+      "value": "debian-8.2-debpkg-9f87c3c-20173003023006"
```

Comment:
@rakeshsinghranchi - why are we using a `debian-8.2` docker image to build the `debian9` deb package?

Looking at https://hub.docker.com/r/microsoft/dotnet-buildtools-prereqs/tags/, I don't see any debian 9 images.  Is this expected?  I think we should be using debian 9 machines here.

/cc @MichaelSimons